### ISSUE TITLE
Modal displayed with attribute aria-hidden="true"

### ISSUE THREAD

**Issue by tambaqui** [react-bootstrap/react-bootstrap#3369]
When the modal is shown, the attribute aria-hidden="true" is being added to all the containers, including the own div role="dialog", which is an error. The modal should have the attribute aria-hidden="false" or nothing.

**Comment by Vavassor** [comment]
I noticed this messing with modal navigation when I was doing screen reader testing for my student project. Does this PR in the react-overlays repo address it? react-bootstrap/react-overlays#293

**Comment by helixbass** [comment]
Yes the issue described here sounds the same as what I ran into and what that PR addresses

The PR has been merged (though afaik a new release including the fix hasn't happened yet) so this issue should be able to be closed
